MealsApp/README.md

```diff
@@ -0,0 +1,48 @@
+
+# MealsApp
+
+
+#### My second Flutter Dummy Project. It is an app to show different meals of different countries and suited for different moods and their pictures and how to cook them with proper ingredients and step by step recipes. 
+
+
+## How to setup on your system :- 
+
+#### Requirements : 
+ 1. Flutter
+ 2. Android Studio 
+ 3. Emulator/Real Device(Android/IOS)
+
+#### How to install : 
+
+1. Check the Official Docs link for installing Flutter : https://flutter.dev/docs/get-started/install 
+2. How to setup the emulator on android studio : https://developer.android.com/studio/run/emulator 
+3. or, connect a physical device : https://stackoverflow.com/questions/54444538/how-do-i-run-test-my-flutter-app-on-a-real-device
+
+
+## Getting Started
+
+Follow these instructions to build and run the project
+
+### Setup Flutter
+
+A detailed guide for multiple platforms setup could be find [here](https://flutter.dev/docs/get-started/install/)
+
+### Setup Project
+
+- Clone this repository using `git clone https://github.com/RohanSengupta326/MealsApp.git`.
+- `cd` into `MealsApp`.
+- `flutter pub get` to get all the dependencies.
+- Generate files using Builder Runner (**required**) 
+```
+flutter packages pub run --no-sound-null-safety build_runner build
+```
+
+## Screenshots
+<p>
+<img src="https://user-images.githubusercontent.com/64458868/135968620-8fe79001-dbac-4c92-af48-62572086a24d.jpg" alt="Splash View" width="200">
+<img src="https://user-images.githubusercontent.com/64458868/135968629-8cd09c81-ed1e-41b3-ba9f-1c383183b545.jpg" alt="Splash View" width="200">
+<img src="https://user-images.githubusercontent.com/64458868/135968631-ecb0bb1d-5ba8-40ef-8bb5-694ecc75f4d2.jpg" alt="Splash View" width="200">
+<img src="https://user-images.githubusercontent.com/64458868/135968632-8c239d21-9b7c-4b54-86b1-aa03749622c3.jpg" alt="Splash View" width="200">
+<img src="https://user-images.githubusercontent.com/64458868/135968633-4cbe61eb-8720-49c1-b774-5d0d50537403.jpg" alt="Splash View" width="200">
```

Comment:
Add the image in the git repo and link here. For example: https://github.com/nisrulz/flutter-examples/tree/develop/dropdown_button

The one you have put is links to Github hosted images.

Reply:
Also please rename them to sc_1, sc_2, sc_3, etc.